docs/dev_guide/index.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Developer guide
+===============
+
+Installing the latest development version
+-----------------------------------------
+
+BayesianTracker has been tested with Python 3.7+ on OS X, Linux and Win10.
```

Comment:
`3.7+` sounds a little too good to be true, has it been tested on `3.10` for example?

Reply:
also has it been tested on M1 mac?

Reply:
That's for a future PR. Create an issue for that. In any case 3.10 should be backwards compatible with 3.7 as far as I know.

Reply:
I know for example the new typing in `3.9` is **not** backwards compatible

Reply:
https://github.com/quantumjot/BayesianTracker/issues/62
https://github.com/quantumjot/BayesianTracker/issues/63

Reply:
:wink: we are not using typing in here yet, so 3.7+ still applies.

Reply:
🤮